bin/suitcss

```diff
@@ -65,7 +65,13 @@ program.parse(process.argv);
 
 var input = program.args[0] ? resolve(program.args[0]) : null;
 var output = program.args[1] ? resolve(program.args[1]) : null;
-var config = program.config ? require(resolve(program.config)) : null;
+var config;
+var configFile = program.config;
+if (configFile && configFile.endsWith('.js')) {
+  config = require(resolve(configFile));
+} else if (configFile) {
+  config = fs.readJsonSync(resolve(configFile));
+}
```

Comment:
sure, functions would also cleanup the cli code a bit - would be helpful also in the importRoot fix, btw. 

I wasnt sure, if you guys wanted this inside the cli code. should these be factored out to something like `cli/utils.js`?